pkgs/development/compilers/vlang/default.nix

```diff
@@ -1,20 +1,20 @@
 { lib, stdenv, fetchFromGitHub, glfw, freetype, openssl, makeWrapper, upx, boehmgc, xorg, binaryen, darwin }:
 
 let
-  version = "weekly.2023.44";
+  version = "0.4.3";
   ptraceSubstitution = ''
     #include <sys/types.h>
     #include <sys/ptrace.h>
   '';
   # Required for bootstrap.
   vc = stdenv.mkDerivation {
     pname = "v.c";
-    version = "unstable-2023-10-30";
+    version = "unstable-2023-12-04";
```

Comment:
I haven't dabbled in vlang for a while.
Is it possible to find and pin the exact commit for this to keep complete compatibility with the release version?

Reply:
> I haven't dabbled in vlang for a while. Is it possible to find and pin the exact commit for this to keep complete compatibility with the release version? I don't think we should use just whatever the latest git version is if possible.

Good call. v.c. is basically the v code auto-ported to C (for bootsrapping), so it should be set to the same version. I've changed it to the correct commit.